beta-src/src/interfaces/PhaseScroll.ts

```diff
@@ -1,5 +1,16 @@
 import { ScrollButtonState } from "../enums/UIState";
 
+interface gameState {
+  seasons: string[];
+  currentSeason: string;
```

Comment:
I don't understand why we have an array of seasons and then are passing in a current season as a string? We only need an index if we are specifying the current season at that point. This would work with a season tuple. Also please enumerate the seasons somewhere.